tests/playwright/sanity/core/app/modules/onboarding/onboarding.test.ts

```diff
@@ -164,3 +164,72 @@ test.describe( 'On boarding @onBoarding', async () => {
 		await expect( kitLibraryTitle ).toBeVisible();
 	} );
 } );
+
+test.describe( 'Onboarding @onBoarding', async () => {
+	test.beforeAll( async ( { browser }, testInfo ) => {
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			e_onboarding: 'active',
+		} );
+	} );
+
+	test.afterAll( async ( { browser }, testInfo ) => {
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			e_onboarding: 'inactive',
+		} );
+	} );
+
+	test( 'Onboarding Choose Features page', async ( { page } ) => {
+		await page.goto( '/wp-admin/admin.php?page=elementor-app#onboarding/chooseFeatures' );
+
+		const chooseFeaturesScreen = page.locator( '.e-onboarding__page-chooseFeatures' ),
+			upgradeNowBtn = page.locator( '.e-onboarding__button-action' ),
+			tierLocator = page.locator( '.e-onboarding__choose-features-section__message strong' ),
+			tiers = {
+				advanced: 'Advanced',
+				essential: 'Essential',
+			};
+
+		await upgradeNowBtn.waitFor();
+
+		await expect.soft( chooseFeaturesScreen ).toHaveScreenshot( 'chooseFeaturesScreen.png' );
+
+		await test.step( 'Check that Upgrade Now button is disabled', async () => {
+			await expect.soft( upgradeNowBtn ).toHaveClass( /e-onboarding__button--disabled/ );
```

Comment:
Question, is `expect.soft()` not only for screenshots?

Reply:
Wasn't completely sure about that.
Fixed.